x-pack/plugins/ingest_manager/server/services/agent_config.ts

```diff
@@ -59,8 +59,13 @@ class AgentConfigService {
         ...DEFAULT_AGENT_CONFIG,
       };
 
-      await this.create(soClient, newDefaultAgentConfig);
+      return await this.create(soClient, newDefaultAgentConfig);
```

Comment:
should be able to remove await here

```suggestion
      return this.create(soClient, newDefaultAgentConfig);
```